### PR TITLE
 Pre-Columbian scenario

### PR DESCRIPTION
**Pull request by jamesmcclain**
This adds a "Pre-Columbian" chart to the "Current Conditions" scenario.

Connects #626

**To Test**
   * Apply the patch
   * Either reprovision the app and the worker, or do a `sudo pip uninstall tr55` `sudo pip install tr55` on both the app and the worker
   * Reload the worker `vagrant reload worker`
   * Go to a project
   * Go to the current conditions scenario.  There should now be a second chart showing the results for the AoI in Pre-Columbian times.

### CODE REVIEW

**Comment by mmcfarland** [comment]
Looks like there was some confusion as to what #626 was asking for.  The stacked chart results for pre-Colombian should just be added to the existing "Current Conditions" scenario.

**Comment by jamesmcclain** [comment]
Oh, okay, I'll add it there.

**Comment by jamesmcclain** [comment]
Okay, I made that change.

File: src/mmw/apps/modeling/tasks.py [hunk 1]
```diff
@@ -211,23 +210,18 @@ def run_tr55(census, model_input):
...
 
-    et_max = 0.207
     precip = _get_precip_value(model_input)
 
     if precip is None:
         raise Exception('No precipitation value defined')
 
-    # TODO: These next two lines are just for
-    # demonstration purposes.
     modifications = get_census_modifications(model_input)
```
Comment (caseypt) [inline src/mmw/apps/modeling/tasks.py]: Should this comment have been removed? I see that the function was slightly modified, but it's still producing fake output.

File: src/mmw/js/src/modeling/tr55/runoff/views.js [hunk 2]
```diff
@@ -46,15 +46,19 @@ var ResultView = Marionette.ItemView.extend({
                     depDisplayNames: ['Infiltration', 'Runoff', 'Evapotranspiration']
                 };
 
-            if (this.scenario.get('is_current_conditions') || this.compareMode) {
+            if (this.compareMode) {
+                data = [
+                    getBarData('', 'modified'),
```
Comment (caseypt) [inline src/mmw/js/src/modeling/tr55/runoff/views.js]: Trailing comma here.

File: src/mmw/apps/modeling/tasks.py (hunk 1 above)
Comment (jamesmcclain) [inline src/mmw/apps/modeling/tasks.py]: No, it should not have been.  I guess I subconsciously remembered making this change twice before (my previous Geotrellis P/Rs)

**Comment by caseypt** [comment]
+1. Tested.

screen shot 2015-08-25 at 9 22 44 am

I'm going to add this to the list of things in #657.

**Comment by lewfish** [comment]
+1, tested.

**Comment by jamesmcclain** [comment]
Okay, many thanks.

**Pull request merged**